assets/src/domain/blocks/i18nInit.ts

```diff
@@ -0,0 +1,4 @@
+import { setLocaleData } from '@wordpress/i18n';
+
+const localeData = window?.eventEspressoData?.i18n;
+setLocaleData(localeData, 'event_espresso');
```

Comment:
In blocks, we need to add text-domain for i18n in order to differentiate between text domains because there are other plugins/themes who register their i18n text domains.

Reply:
cool, didn't knew this